main.tf

```
@@ -27,7 +27,7 @@ resource "aws_codestarnotifications_notification_rule" "pipeline_updates" {
   count          = length(var.codepipelines)
   detail_type    = "FULL"
   event_type_ids = var.event_type_ids
-  name           = "slackNotification${var.codepipelines[count.index].name}"
+  name           = join("-", ["slackNotification${var.codepipelines[count.index].name}", module.default_label.name])
```

Since we are changing the name here anyway, I wonder if we should restructure it like this:
```suggestion
  name           = join("-", [module.default_label.name, var.codepipelines[count.index].name])
```
Was not a big fan of that static prefix and now that we are making the name longer, the 64 character limit here is starting to worry me.

---

Agree - I've done a small refactor along these lines. Let me know what you think.